src/main/java/seedu/address/logic/commands/viewcommand/ViewCommand.java

```diff
@@ -12,9 +12,10 @@
 public abstract class ViewCommand extends Command {
 
     public static final String COMMAND_WORD = "view";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": shows details of the entity with the specified ID. "
-            + "Parameters: mentor/participant/team ID\n"
-            + "Example to view Mentor with ID M-1: " + COMMAND_WORD + " M-1";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": shows details of the specified "
+            + "entity with the specified ID. "
+            + "Format: view [entity name] [entity ID] \n"
```

Comment:
same  here

Reply:
Oh yea good catch. I have changed it now. Thanks !